store/src/process_records.rs

```diff
@@ -306,6 +311,10 @@ pub fn process_record(
     }
   }
 
+  if let Some(c) = comment {
```

Comment:
use `trim` & `isEmpty` instead of `match`

Reply:
fixed